### ISSUE TITLE
After seeking, PlayerState becomes "buffering" when a song finished playing.

### ISSUE THREAD

**Issue by dylancom** [tumtumtum/StreamingKit#457]
When just playing a song from beginning to end, the **STKAudioPlayerState** will change into **STKAudioPlayerStateStopped**.
But when I seek during playback (seeking works fine btw) for some reason the state will change from **STKAudioPlayerStatePlaying** to **STKAudioPlayerStateBuffering** when the song finished playing.

**Comment by DougFischer** [comment]
Hi @dylancom! I'm studying this lib to adopt it in my next project and I'm interested if this issue is already addressed. May you share if you ended up using this lib?

**Comment by dylancom** [comment]
@DougFischer Yes I'm still using this lib. This issue hasn't been solved but I found a workaround by comparing the progress of a song with the duration. Not sure if this stuck at "buffering" problem happens with all audio streams though.
